content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -141,6 +141,80 @@ limit.
 On Linux, resource allocations for Pod level control groups (cgroups) are based on the effective Pod
 request and limit, the same as the scheduler.
 
+### Adopting sidecar containers
```

Comment:
This section might be more suitable for a reference page rather than a section of the concepts page.

Reply:
I would appreciate a pointer on which page is better. I didn't think it deserve the whole task page under https://kubernetes.io/docs/tasks/. I am not sure where under reference would be a good place.